.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "jasmine": true
+  },
+  "globals": {
+    "require": true
+  },
+  "rules": {
+    "one-var": 0,
+    "one-var-declaration-per-line": 0,
+    "new-cap": 0,
+    "consistent-return": 0,
+    "no-param-reassign": 0,
+    "class-methods-use-this": 0,
+    "no-prototype-builtins": 0,
+    "no-bitwise": 0,
+    "no-trailing-spaces": 0,
+    "arrow-body-style": ["error", "always"],
+    "comma-dangle": 0,
+    "curly": ["error", "multi-line"],
+    "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+    "valid-jsdoc": ["error", {
+      "requireReturn": true,
+      "requireReturnType": true,
+      "requireParamDescription": false,
+      "requireReturnDescription": true
+    }],
+    "require-jsdoc": ["error", {
+        "require": {
+            "FunctionDeclaration": true,
+            "MethodDefinition": true,
+            "ClassDeclaration": true
+        }
+    }]
+  }
+}
```

Comment:
**Nitpick**
Please, use [editorConfig](http://editorconfig.org/) to remove `EOF`